frontend/src/pages/Home/Llama.js

```diff
@@ -64,9 +64,6 @@ export default function Llama({ user }) {
   }, [user.id]);
 
   const addWiggler = () => {
-    if (!llamaImage.current) {
```

Comment:
unnecessary, removed to boost test coverage

Reply:
👍